extensions/amp-reach-player/0.1/amp-reach-player.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {loadPromise} from '../../../src/event-helper';
+
+class AmpReachPlayer extends AMP.BaseElement {
+
+  /** @override */
+  createdCallback() {
+    this.preconnect.url('https://player-cdn.beachfrontmedia.com');
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return isLayoutSizeDefined(layout);
+  }
+
+  /** @override */
+  layoutCallback() {
+
+    const width = this.element.getAttribute('width');
+    const height = this.element.getAttribute('height');
+    const embedId = (this.element.getAttribute('data-embed-id') || 'default');
+    const iframe = document.createElement('iframe');
+
+    iframe.setAttribute('frameborder', 'no');
```

Comment:
Unfortunately not. Most of our players have dynamic content, and until the player makes a content request, it is not know what that content will be.

I could create a generic one if needed, but it would not reflect the content the will play.